app/components/Card/CardFront.tsx

```diff
@@ -74,7 +70,7 @@ export default function CardFront({
 
 const responsiveValues = {
   carousel: {
-    size: { md: "front_md", "2xl": "front_lg" },
+    size: { xs: "front_xs", sm: "front_sm", md: "front_md", "2xl": "front_lg" },
```

Comment:
See my comment for the CardBack component. 
```suggestion
    size: { base: "front_xs", sm: "front_sm", md: "front_md", "2xl": "front_lg" },
```
Again, you'll have to add sizing for "base" and "sm" breakpoints for lines 75-79. You will also need to add new sizing for MachineTypeLabel component for "base" and "sm" breakpoints if you want to use the new card sizes. Lmk if you need my help with that, because it might be a bit tedious and finicky.

I made the same mistake here with setting "md" breakpoint instead of "base" breakpoint on lines 73-79. I have the same issue within MachineTypeLabel component.

Reply:
I played around with it a bit, changed all the first sizes to "base" and updated the MachineTypeLabel responsive values. It was finicky and I may not have gotten it perfectly, but you can let me know what you think.